pycroft/lib/user.py

```diff
@@ -133,15 +134,18 @@ def store_user_sheet(new_user, plain_password, timeout=15, generation_purpose=''
     :param str plain_password:
     :param str generation_purpose:
     :param int timeout: The lifetime in minutes
+    :param bool wifi: Generate a user sheet for wifi access instead of the generic one
     """
-    pdf_data = b64encode(generate_user_sheet(new_user, plain_password, generation_purpose)).decode('ascii')
+    if wifi is True:
```

Comment:
`is True` is an antipattern, because this breaks python's duck-typing. Correct would be just `if wifi` – then, one could pass other values (such as a mock) which evaluate to `True` under the lense of `__bool__`, `__len__`, etc.

cf
- [__bool__ (python3 docs)](https://docs.python.org/3/reference/datamodel.html#object.__bool__)
- [PEP8, Programming recommendations, 21st point](https://www.python.org/dev/peps/pep-0008/#programming-recommendations)